### PR TITLE
Added MX Record and TXT Record

### PR DESCRIPTION
**Pull request by iostpa**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [ ] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by CuteDog5695** [comment]
github.io domain doesnt work because i changed it to is-a.dev, i just made this so that i can get email domain

File: domains/cutedog5695.json [hunk 1]
```diff
@@ -6,6 +6,11 @@
         "twitter": "cutedog5695"
     },
     "record": {
-        "CNAME": "cutedog5695.github.io"
-    }
+        "CNAME": "cutedog5695.github.io",
```
Comment (DEV-DIBSTER) [inline domains/cutedog5695.json:9]: CNAME cannot be used in conjunction with other records and vice versa.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/cutedog5695.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/cutedog5695.json:9]: so basically i cant use CNAME if i want to add other records?

**Comment by CuteDog5695** [comment]
closing pull request, ill stay with just domain for site for now

**Pull request closed without merging**